app/SharedExpense.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SharedExpense extends Model
```

Comment:
Oh you're talking about adding an instance variable inside expense? Makes sense. Sorry I was thinking of DAO's again rather than models.